data/en/cffunction.json

```diff
@@ -10,7 +10,7 @@
 		{"name":"returntype","description":"String; a type name; data type of the function return value. Required for a web service; optional otherwise.\n","required":false,"default":"any","type":"string","values":["any","array","binary","boolean","date","guid","numeric","query","string","struct","UUID","variablename","void","xml","(component name)"]},
 		{"name":"roles","description":"A comma-delimited list of CFML security roles that can invoke the method. Only users who are logged in with the specified roles can execute the function. If this attribute is omitted, all users can invoke the method.","required":false,"default":"","type":"string","values":[]},
 		{"name":"access","description":"The client security context from which the method can be invoked.\n\n private: Available only to the component that declares the method and any components that extend the component in which it is defined.\n\n package: Available only to the component that declares the method, components that extend the component, or any other components in the package.\n\n public: Available to a locally executing page or component method.\n\n remote: Available to a locally or remotely executing page or component method, or a remote client through a URL, Flash, or a web service. To publish the function as a web service, this option is required.\n","required":false,"default":"public","type":"string","values":["private","package","public","remote"]},
-		{"name":"output","description":"Specifies under which conditions the function can generate HTML output.\n","required":false,"default":"","type":"boolean","values":[true,false]},
+		{"name":"output","description":"Specifies under which conditions the function can generate HTML output.\nLucee These values can be defined by "Yes" for true or "No" for false.\n","required":false,"default":"","type":"boolean","values":[true,false]},
```

Comment:
This should be part of a separate PR.

But also, this change is not right. First, the syntax is wrong and breaking the JSON. The `"`s need to be escaped in a JSON string. Second, using "Yes" and "No" for booleans should work in all CFML engines, not just Lucee. The change that should be made here is removing `true` and `false` from `values` since that should only be used for enumerated values.

Reply:
I apologize. I was testing with the documentation on the cffunction page today. and trying the out CF and Lucee notations. At the time, I noticed that Lucee documentation made a mention of only `Yes` and `No` as options, so I was trying it out. I failed to realize that would effect the pull request at the time. I also received the error in JSON, and I plan to fix it. For now, I have removed that addition and will look more into branches and local testing.